public/slack-app-registry.json

```diff
@@ -13,11 +13,16 @@
                 "description" : "Returns a random xkcd comic.",
                 "manual" : "THERE IS NO DOCUMENTATION WEBSITE. MAKE YOUR OWN."                
             },
-	    "lowPriority" : {
-		"usage" : "/struggle lowPriority",
-		"description" : "Mark has no idea what this function does.",
-		"manual" : "Hope Brett doesn't kill us."
-	    }
+            "lowPriority" : {
+                "usage" : "/struggle lowPriority",
+                "description" : "Mark has no idea what this function does.",
+                "manual" : "Hope Brett doesn't kill us."
+            },
+            "inviteme" : {
+                "usage" : "/struggle dev invite",
```

Comment:
Sorry, I recommended changing it to `/struggle dev invite`, but this would require making a `dev` named function first. So, just keep it to `inviteme` for now, and change this to, `/struggle inviteme`.